src/lib.rs

```diff
@@ -25,7 +25,7 @@ impl Default for Params {
             num_segments: 12,
             distances: vec![1.0, 3.0, 6.0, 10.0, 15.0],
             // num_vertices: 121,
```

Comment:
Having commented-out code is useful during the learning process, but when it builds up too much, it just makes things harder to read. This is redundant, now that we're using 121 again.

Reply:
:+1: removed.